rpcs3/Emu/Cell/Modules/cellCamera.cpp

```diff
@@ -359,6 +379,11 @@ s32 cellCameraIsAvailable(s32 dev_num)
 {
 	cellCamera.todo("cellCameraIsAvailable(dev_num=%d)", dev_num);
 
+	if (g_cfg.io.camera == camera_handler::null)
+	{
+		return false;
```

Comment:
Why not CELL_OK? They both represent 0

Reply:
Why? This is more correct semantically. You're relying on the fact that `CELL_OK == 0` for no reason.
Also it's consistent with the rest of the file.